Robinhood/Robinhood.py

```diff
@@ -73,11 +56,75 @@ class Robinhood:
     logger = logging.getLogger('Robinhood')
     logger.addHandler(logging.NullHandler())
 
+    from . import exceptions
+
+    class Bounds(Enum):
+        """enum for bounds in `historicals` endpoint"""
+        REGULAR = 'regular'
+        EXTENDED = 'extended'
+
+    class Transaction(Enum):
+        """enum for buy/sell orders"""
+        BUY = 'buy'
+        SELL = 'sell'
+
+    class Trigger(Enum):
+        """enum for buy/sell orders"""
+        IMMEDIATE = 'immediate'
+        STOP = 'stop'
+
+    class Order(Enum):
+        """enum for buy/sell orders"""
+        MARKET = 'market'
+        LIMIT = 'limit'
+
+    class TimeForce(Enum):
+        """enum for buy/sell orders"""
+        GTC = 'gtc'
+        GFD = 'gfd'
+        IOC = 'ioc'
+        FOK = 'fok'
+        OPG = 'opg'
+
+    class OrderState(Enum):
+        """enum for order states"""
+        QUEUED = "queued"
+        UNCONFIRMED = "unconfirmed"
+        CONFIRMED = "confirmed"
+        PARTIALLY_FILLED = "partially_filled"
+        FILLED = "filled"
+        REJECTED = "rejected"
+        CANCELLED = "cancelled"
+        FAILED = "failed"
+
+        def stopped(self):
+            return self.name in ('FILLED', 'CANCELLED', 'REJECTED', 'FAILED')
+        def active(self):
+            return self.name in ('QUEUED', 'CONFIRMED', 'PARTIALLY_FILLED')
+        def other(self):
+            return self.name in ('UNCONFIRMED')
 
     ###########################################################################
     #                       Logging in and initializing
     ###########################################################################
 
+    def raiseForStatus(self,res):
```

Comment:
would prefer `_raise_for_status` as func name.  Camel-case is for classes, and `_` would denote "internal"

Reply:
ok